src/test/java/com/mongodb/kafka/connect/sink/SinkTestHelper.java

```diff
@@ -40,12 +41,20 @@
 
     public static  Map<String, String> createConfigMap(final String json) {
         Map<String, String> map = createConfigMap();
-        Document.parse(json).forEach((k, v) -> map.put(k, v.toString()));
+        Document.parse(json).forEach((k, v) -> {
+            if (k.equals(TOPICS_REGEX_CONFIG)) {
+                map.remove(TOPICS_CONFIG);
```

Comment:
By default TOPICS_CONFIG was always added in `createConfigMap` - this just ensures that its removed when the regex config is set.